ravem/tests/util_test.py

```diff
@@ -6,15 +6,13 @@
 # the LICENSE file for more details.
 
 import pytest
+from indico_ravem.plugin import RavemPlugin
+from indico_ravem.util import has_access, ravem_api_call
```

Comment:
the old place for those imports were correct. if isort moved them here something with the config is wrong

Reply:
I was pointing the wrong config file. I've updated all the files.